Asterix/tests/test_wfsc.py

```diff
@@ -0,0 +1,118 @@
+import os
+import shutil
+import pytest
+import numpy as np
+
+from Asterix.main_THD import THD2
+from Asterix import Asterix_root
+from Asterix.utils import get_data_dir, read_parameter_file
+from Asterix.wfsc import Estimator, Corrector, MaskDH, correction_loop
+
+
+def quick_run_no_save(config, data_dir):
```

Comment:
You might not need this. I think you can just define a test function where you pass `tmpdir`, a pytest ficture that creates a throw-away path for output files for you, and then you can use that as the output path in your tests. The function name would look something like this:
```python
def test_a_wfsc_run(tmpdir):
```
and then you use `tmpdir` as your path.

I haven't tested this for Asterix though, there could be some hickups, but if it works it would be much easier than defining a whole new function.

Reply:
Sorry I don't understand. 
I create a new functions so that I can separatly call it for 1 or 2 dm correction